### PR TITLE
Update onboard-to-stripe-in-the-merchant-portal.md

### PR DESCRIPTION
**Pull request by matweew**
## PR Description

Added info box to check `redirect.php` file.

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/payment-service-provider/202404.0/marketplace/stripe-third-party-integration/onboard-to-stripe-in-the-merchant-portal.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ To onboard to Stripe as a merchant, follow the steps:
 3. To complete the onboarding, follow the prompts to fill in merchant information.
   Once you finish onboarding, the **Payment Settings** page is opened. Stripe needs to approve your onboarding request before you can start using it. You can check the status of onboarding in the Stripe dashboard.
 
+{% info_block infoBox "" %}
+
+Please make sure that your Merchant Portal has `redirect.php` in the web server public folder: [`/public/MerchantPortal/redirect.php`](https://github.com/spryker-shop/b2c-demo-marketplace/blob/master/public/MerchantPortal/redirect.php).
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment-service-provider/202404.0/marketplace/stripe-third-party-integration/onboard-to-stripe-in-the-merchant-portal.md:18]: @matweew Why is this information in this document and why is this file needed?

Comment (matweew) [reply docs/pbc/all/payment-service-provider/202404.0/marketplace/stripe-third-party-integration/onboard-to-stripe-in-the-merchant-portal.md:18]: The reason:
 The file is needed to allow redirects from 3rd party sites to the Merchant Portal. The default session cookie settings is SameSite=Strict and the browser does not allow to read session cookies.
 Using the intermediate page on the Merchant Portal domain, the session cookies are visible and a user can continue with his session.


Why:
1. because it's the doc for Merchant Portal. So to emphasize it's related to the process of Merchant Onboarding when merchants are redirected from Stripe back to Merchant Portal.
2. We can also update the installation guide https://docs.spryker.com/docs/pbc/all/payment-service-provider/202404.0/marketplace/stripe-third-party-integration/install-and-configure-stripe-prerequisites-for-marketplace.html (but there it could be missed and later would be not clear why redirect is not working)
